.github/workflows/sonar-scanner-cpp.yml

```diff
@@ -7,7 +7,7 @@ on:
         description: 'Build image tag for ghcr.io/usdot-fhwa-stol/carma-builds'
         required: false
         type: string
-        default: 'latest'
```

Comment:
Latest image pulls a random image since latest is currently which ever of the matrix builds that finishes last